system-test/storage.ts

```diff
@@ -780,6 +780,120 @@ describe('storage', () => {
     });
   });
 
+  describe('public access prevention', () => {
+    let bucket: Bucket;
```

Comment:
How are the bucket names for these tests generated? Will they use the same bucket or separate buckets?

Just want to make sure we don't set ourselves up for tests that are flaky or need to run in a particular order.

Reply:
These tests are using the same bucket. Good catch! I will make this change.